### PR TITLE
Aiohttp

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

**Comment by GuillaumeDSM** [comment]
up with input url

File: octobot_commons/aiohttp_util.py [hunk 1]
```diff
@@ -14,6 +14,18 @@
...
+import ssl
+
+import contextlib
+import aiohttp
+import certifi
+
+import octobot_commons.logging
+import octobot_commons.constants
+
+DEFAULT_TEST_URL = (
```
Comment (Herklos) [inline octobot_commons/aiohttp_util.py:26]: Should we move it in tests as it is only used in tests ?

Comment (GuillaumeDSM) [reply octobot_commons/aiohttp_util.py:26]: It's used by octobot and tentacles manager as well

**Comment by GuillaumeDSM** [comment]
up

**Review by Herklos — APPROVED** [review]

**Pull request merged**